app/models/ex_link.rb

```diff
@@ -0,0 +1,4 @@
+class ExLink < ActiveRecord::Base
+  validates :label, presence: true
```

Comment:
The validates-method have some nice syntax, this can be changed too:
`validates :label, :url, presence: true`